### PR TITLE
Forms section migration and consolidation

### PR DESCRIPTION
**Pull request by esthermmoturi**
One of 2 PRs to close https://github.com/medic/cht-docs/issues/1639

This PR is part of the docsite revamp work, the idea is to consolidate CHT Concepts into one section. In this case, the PR consolidated documentation on what CHT forms are , how to configure and maintain them.

All files will be placed under Building/Forms folder. Here are the items that will be moved:

- [Building/Concepts/Forms](https://docs.communityhealthtoolkit.org/building/concepts/forms/)
- [Building/References/Forms](https://docs.communityhealthtoolkit.org/building/reference/forms/)
- [Building/Guide/Forms/](https://docs.communityhealthtoolkit.org/building/guides/forms/)
- [Building/Tutorials/App forms](https://docs.communityhealthtoolkit.org/building/tutorials/app-forms/)
- [Building/Tutorials/Form properties](https://docs.communityhealthtoolkit.org/building/tutorials/form-properties/)

### CODE REVIEW

**Review by andrablaj — COMMENTED** [review]
@esthermmoturi I only had the time to review a few files. I will be off for a few days, so I will remove myself as a reviewer and suggest you add another team member. Sorry for that!

**Comment by esthermmoturi** [comment]
Thank you for the suggestions @andrablaj . No worries. I will assign somebody else. Enjoy your time off!

File: content/en/building/cht-forms/_index.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ Forms are a building block of all CHT apps. They are used when creating or editi
...
-- [**App Forms**]({{< ref "building/reference/forms/app" >}}): serve as actions within the app, such as a task or an action. Defined as CHT-enhanced XForms.
-- [**Collect Forms**]({{< ref "building/reference/forms/collect" >}}): used to render forms in Medic Collect. Defined as ODK XForms and need a corresponding JSON form to receive data in CHT.
+- [**Contact Forms**]({{< ref "building/cht-forms/forms/contact" >}}): used to create and edit contacts. Defined as CHT-enhanced XForms.
+- [**App Forms**]({{< ref "building/cht-forms/forms/app" >}}): serve as actions within the app, such as a task or an action. Defined as CHT-enhanced XForms.
+- [**Collect Forms**]({{< ref "building/cht-forms/forms/collect" >}}): used to render forms in Medic Collect. Defined as ODK XForms and need a corresponding JSON form to receive data in CHT.
 - [**JSON Forms**]({{< ref "building/reference/app-settings/forms" >}}): used for data coming from external channels such as SMS, or via interoperability with other tools. Defined according to a JavaScript Object Notation schema.
```
Comment (mrjones-plip) [inline content/en/building/cht-forms/_index.md:25]: as this is in a new section, and is actually about configuring forms, not about buildig forms, I think this shouldn't be here.  we have in related content already

```suggestion
```

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
This PR nor the [upstream ticket](https://github.com/medic/cht-docs/issues/1639) are not clear to an outside reviewer what the goal is.  I think it's to move `building/reference/forms/` to  `building/cht-forms/forms/` ?  Please add more details to the ticket and the PR.  Here's an [example PR](https://github.com/medic/cht-docs/pull/1673) that has a video demonstrating the change (pretty involved!). A more simple approach would be [to list out the changes](https://github.com/medic/cht-docs/pull/1618) in the PR.  As well, the ticket should have enough details that folks know what the change is about.

I assume we a already decided on the `building/cht-forms/forms` path?  I find it confusing and think we should consider changing it.  This is because on the current site we have this hierarchy:

```
Building            (building)
    Reference       (building/reference)
        forms/      (building/reference/forms)
            app     (building/reference/forms/app)
            collect (building/reference/forms/collect)
            contact (building/reference/forms/contact)
```

In this PR as is, we want to do this:

```
Building            (building)
    Forms           (building/cht-forms)
        forms       (building/cht-forms/forms)
            app     (building/cht-forms/forms/app)
            collect (building/cht-forms/forms/collect)
            contact (building/cht-forms/forms/contact)
```

The use of the `/cht-forms` URL and directory seems superfluous (already in the `forms` directory, we don't need another `forms` directory) and confusing (are there other non-cht forms?) .  I propose we flatten it.  While we're in there we can rename both `Form Properties` and `Versioning forms`.  This presents a very logical hierarchy without the repetitious use of `forms`: 


```
Building            (building)
    Forms           (building/forms)
        app         (building/forms/app)
        collect     (building/forms/collect)
        contact     (building/forms/contact)
        Properties  (building/forms/properties)
        Versioning  (building/forms/versioning)
```

**Comment by esthermmoturi** [comment]
I have left the .md files named as is(form-properties.md and versioning.md) but changed the Title that appears in the site. This is what we expect to see:
image

**Comment by mrjones-plip** [comment]
@esthermmoturi - I'll try and get to this in the next couple of days!

File: content/en/building/cht-forms/_index.md (hunk 1 above)
Comment (mrjones-plip) [reply content/en/building/cht-forms/_index.md:20]: This bullet item is still in this file.  unresolving to track the request

(and if you don't want to do this, lemme know why)

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
Thanks for doing a round of fixes!  I think maybe one change got reverted?  I unresolved it for you to follow up.

Otherwise two other changes please:

1. while this URL is being created let's move it from `/building/cht-forms/` -> `/building/forms/` .  One less CHT is better!

2. there's a number of `404`s this PR creates.  Please use the checker script to itterate and ensure they're all fixed:

```
http://localhost:1313/building/guides/forms/additional-docs/ 404
http://localhost:1313/building/guides/forms/app-form-sms/ 404
http://localhost:1313/building/guides/forms/form-inputs/ 404
http://localhost:1313/building/guides/forms/google-drive/ 404
http://localhost:1313/building/guides/forms/multimedia/ 404
http://localhost:1313/building/guides/forms/report-titles/ 404
http://localhost:1313/building/guides/forms/uhc-mode/ 404
http://localhost:1313/building/guides/forms/wealth-quintiles/ 404
http://localhost:1313/building/reference/forms/ 404
http://localhost:1313/building/reference/forms/app/ 404
http://localhost:1313/building/reference/forms/collect/ 404
http://localhost:1313/building/reference/forms/contact/ 404
```

**Comment by esthermmoturi** [comment]
The script is giving 200 as follows:
```
http://localhost:1313/building/guides/forms/additional-docs/ 200
http://localhost:1313/building/guides/forms/app-form-sms/ 200
http://localhost:1313/building/guides/forms/form-inputs/ 200
http://localhost:1313/building/guides/forms/google-drive/ 200
http://localhost:1313/building/guides/forms/multimedia/ 200
http://localhost:1313/building/guides/forms/report-titles/ 200
http://localhost:1313/building/guides/forms/uhc-mode/ 200
http://localhost:1313/building/guides/forms/versioning/ 200
http://localhost:1313/building/guides/forms/ 200
http://localhost:1313/building/guides/forms/wealth-quintiles/ 200
```

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
Can you have another look at the `404`s? Between the time I reviewed this PR and now, you made two commits ([1 ](https://github.com/medic/cht-docs/pull/1696/commits/899a06246f4870e91ccfeefd117ef38080dc9d80)and [2](https://github.com/medic/cht-docs/pull/1696/commits/882e6b73aa743fd4a58eab31f94a8498eccfb7b2)) and neither has any additions to `aliases:` section.  I'm seeing all the `404`s I found as still needing to be fixed.

As well, while you're in there, can you move `/building/forms/configuring-forms/` to be just `/building/forms/configuring/` ?  I should have noticed this from the start, but the number of `forms` in the URL seems overkill (we don't need to fix the existing URLs, only the new ones I think!):

  /building/**forms**/configuring-**forms**/**form**-inputs/

**Comment by esthermmoturi** [comment]
The standard naming for all other sections is configuring-xxx(example [here](https://github.com/medic/cht-docs/blob/main/content/en/building/translations/configuring-translations.md) ), this makes it easier to identify files other than each section having a file named `configuring` so let's leave it as is.

Aliases were not added since the files had not been merged to main with new names/links.(for commit 1, commit 2 omitted a line within the form). I pushed commits to address feedback provided within the PR.

**Comment by mrjones-plip** [comment]
yay! `404`s all look good!  

If we can tend to the extra `forms` in the URL, I think we'll be good:

>   As well, while you're in there, can you move `/building/forms/configuring-forms/` to be just `/building/forms/configuring/` ?  I should have noticed this from the start, but the number of `forms` in the URL seems overkill (we don't need to fix the existing URLs, only the new ones I think!):
>
>  /building/**forms**/configuring-**forms**/**form**-inputs/

**Review by mrjones-plip — APPROVED** [review]
looks good!  Thanks for all the follow up to see this through.

**Pull request merged**